root/static/scripts/relationship-editor/components/RelationshipEditor.js

```diff
@@ -477,10 +480,6 @@ export function runReducer(
       /*:: exhaustive(action); */
     }
   }
-
-  if (__DEV__) {
-    deepFreeze(writableState);
-  }
```

Comment:
Since `runRelationshipEditorReducer` is now called in a loop, I had to move this to `reducer`.  This just makes sure we're not performing direct writes to any state (and is only enabled in development).